utils/rules-template.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Rules related to {{{ component }}} on {{{ product }}}</title>
+<style>
+</style>
+</head>
+<body>
+
+<h1>Rules related to {{{ component }}}</h1>
+
+<div class="rules">
+{{% for rule in rules -%}}
+<div class="one_rule">
+	<h2>{{{ loop.index }}}. {{{ rule.title }}}</h2>
+  <p>{{{ rule.description | safe }}}</p>
```

Comment:
I noticed one issue when rendering a rule description that references a variable, for example [accounts_password_pam_difok](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_difok/rule.yml#L7).

It doesn't render any text as the variable's default value, and may cause any text to be subscript.
```html
The pam_pwquality module's <tt>difok</tt> parameter sets the number of characters
in a password that must not be present in and old password during a password change.
<br><br>
Modify the <tt>difok</tt> setting in <tt>/etc/security/pwquality.conf</tt>
to equal <sub idref="var_password_pam_difok"> to require differing characters
when changing passwords.</sub>
```